src/models/pang13/repairs.md

```diff
@@ -0,0 +1,178 @@
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 11 bottom panel screws.
+    - The [M.2 access door screw](#replacing-an-m2nvme-ssd) does not need to be removed.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Lift the bottom panel off, starting from the hinges in the back.
+
+## Removing the battery:
+
+The battery provides primary power whenever the system is unplugged.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove the four battery screws, highlighted green below.
+
+![Battery screws](./img/battery.webp)
+
+3. Unplug the battery connector (highlighted cyan above) and lift the battery out of the chassis.
+
+## Replacing the wireless card:
+
+Your Pangolin's WiFi and Bluetooth are both handled by the same module. It is a standard M.2 2230 slot with PCIe and USB interfaces (A Key).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 12 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the WiFi/Bluetooth module:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Locate the wireless module. Remove any clear tape that is securing the wires.
+
+![Wireless card](./img/wireless-card.webp)
+
+3. Gently remove the two antennas (highlighted red above) by pulling them up and away from the wireless card.
+4. Remove the retaining screw opposite the M.2 slot, highlighted yellow above.
+5. The wireless card will pop up at an angle. Remove the card from the M.2 slot.
+6. Insert the new wireless card into the M.2 slot at an angle.
+7. Replace the retaining screw.
+8. Attach the two antennas by aligning the circular fittings and pressing onto the wireless card. The connectors will snap into place. _Use caution when attaching the connectors; the pins can bend, break, or snap._
+
+## Replacing the CMOS battery:
+
+The CMOS battery supplies power to the system's CMOS chip. UEFI settings and the computer's hardware clock are stored on the CMOS. If your system doesn't boot, you can reset the CMOS to force a low-level hardware reset. If your clock is constantly resetting, it's likely your CMOS battery needs to be replaced.
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 10 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the CMOS battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Unplug the white connector attaching the CMOS battery to the motherboard (highlighted green below.)
+
+![CMOS battery](./img/cmos-battery.webp)
+
+3. If you are replacing the CMOS battery, carefully peel it away from the adhesive holding it to the motherboard.
```

Comment:
Adhesive is holding the battery to the chassis, not  the motherboard, right?

Reply:
Looks like it to me. Corrected for both pang12 and pang13 with a693974.